R/utils-vegaspec.R

```diff
@@ -81,6 +81,13 @@
   x
 }
 
+# Method for vegaspec
+.find_urls <- function(spec){
```

Comment:
Yeah perhaps this could be made safer by looking for 'data.url' as the name in the list?  When unlist operates on a nested list, the names end up like `outermost.middle.innermost` so if 'url' should always be nested one down in 'data' the regex could change to check for that?